DB/src/main/java/io/deephaven/db/tables/utils/csv/InferenceSpecs.java

```diff
@@ -0,0 +1,294 @@
+     * <li>{@link Parser#BOOL}</li>
+     * <li>{@link Parser#CHAR}</li>
+     * <li>{@link Parser#BYTE}</li>
+     * <li>{@link Parser#FLOAT}</li>
+     * </ul>
+     *
+     * Uses the default {@link #onNullParser()}.
+     *
+     * <p>
+     * Note: the non-string parsers are only relevant when the appropriate {@link #limitToType(Type)} is invoked.
+     *
+     * @return the string-only inference
+     */
+    public static InferenceSpecs strings() {
+        return builder().addParsers(
+                Parser.STRING,
+                Parser.INSTANT,
+                Parser.SHORT,
+                Parser.INT,
+                Parser.LONG,
+                Parser.DOUBLE,
+                Parser.BOOL,
+                Parser.CHAR,
+                Parser.BYTE,
+                Parser.FLOAT)
+                .build();
+    }
+
+    /**
+     * The "minimal" inference.
+     *
+     * <p>
+     * Contains the following parsers:
+     *
+     * <ul>
+     * <li>{@link Parser#INSTANT}</li>
+     * <li>{@link Parser#LONG}</li>
+     * <li>{@link Parser#DOUBLE}</li>
+     * <li>{@link Parser#BOOL}</li>
+     * <li>{@link Parser#STRING}</li>
+     * <li>{@link Parser#BYTE}</li>
+     * <li>{@link Parser#SHORT}</li>
+     * <li>{@link Parser#INT}</li>
+     * <li>{@link Parser#FLOAT}</li>
+     * <li>{@link Parser#CHAR}</li>
+     * </ul>
+     *
+     * Uses the default {@link #onNullParser()}.
+     *
+     * <p>
+     * Note: the byte, short, int, float, and char parsers are only relevant when the appropriate
+     * {@link #limitToType(Type)} is invoked.
+     *
+     * @return the minimal inference
+     */
+    public static InferenceSpecs minimal() {
+        return builder().addParsers(
+                Parser.INSTANT,
+                Parser.LONG,
+                Parser.DOUBLE,
+                Parser.BOOL,
+                Parser.STRING,
+                Parser.BYTE,
+                Parser.SHORT,
+                Parser.INT,
+                Parser.FLOAT,
+                Parser.CHAR)
+                .build();
+    }
+
+    /**
+     * The "standard" inference, does not parse floats or bytes.
+     *
+     * <p>
+     * Contains the following parsers:
+     *
+     * <ul>
+     * <li>{@link Parser#INSTANT}</li>
+     * <li>{@link Parser#SHORT}</li>
+     * <li>{@link Parser#INT}</li>
+     * <li>{@link Parser#LONG}</li>
+     * <li>{@link Parser#DOUBLE}</li>
+     * <li>{@link Parser#BOOL}</li>
+     * <li>{@link Parser#CHAR}</li>
+     * <li>{@link Parser#STRING}</li>
+     * <li>{@link Parser#BYTE}</li>
+     * <li>{@link Parser#FLOAT}</li>
+     * </ul>
+     *
+     * Uses the default {@link #onNullParser()}.
+     *
+     * <p>
+     * Note: the byte and float parsers are only relevant when the appropriate {@link #limitToType(Type)} is invoked.
+     *
+     * @return the standard inference
+     */
+    public static InferenceSpecs standard() {
+        return builder().addParsers(
+                Parser.INSTANT,
```

Comment:
Should INSTANT_DB be here?

Reply:
I'd like to get away from old instant formats... that said, our *default* inference is `standardTimes()` which includes it...